app/(protected)/home/index.tsx

```diff
@@ -34,12 +45,36 @@ export default function Home() {
           <SummaryCard value={5000} type="income" />
           <SummaryCard value={2000} type="outcome" />
         </View>
-        <DatePicker value={date} onChange={(data) => {
-        setDate(data);
-      }}
-                  label="Data">
-      </DatePicker>
-      <View style={styles.transactions}>
+        <DateRangeSelector
+          onDateChange={(start, end) => setTransactionDate({ start, end })}
+          open={showDateFilter}
+          onClose={() => setShowDateFilter(false)}
+        />
+        <OptionsSelector
+          open={showTransactionFilter}
+          onClose={() => setShowTransactionFilter(false)}
+          handleOptionSelection={(option) => setTransactionFilter(option)}
+        />
+
+        <View style={styles.filtersContainer}>
+          <TouchableOpacity
+            onPress={() => setShowDateFilter(true)}
+            style={styles.filterButton}
+          >
+            <Text style={styles.filterButtonText}>Período</Text>
+            <MaterialIcons name="arrow-drop-down" size={24} color="black" />
+          </TouchableOpacity>
+
+          <TouchableOpacity
+            onPress={() => setShowTransactionFilter(true)}
+            style={styles.filterButton}
+          >
+            <Text style={styles.filterButtonText}>Movimentações</Text>
+            <MaterialIcons name="arrow-drop-down" size={24} color="black" />
+          </TouchableOpacity>
+        </View>
```

Comment:
Nós já temos a tela de Extrato, planejamos adicionar esses filtros lá também? Se for essa a ideia, acho que seria legal já extrair esse trecho dos filtros para um componente separado para reutilizarmos nas duas telas, o que acha?

Reply:
Boa!
Vou fazer isso.